sklearn/multiclass.py

```diff
@@ -75,6 +75,15 @@ def _fit_binary(estimator, X, y, classes=None):
     return estimator
 
 
+def _partial_fit_binary(estimator, X, y, all_classes=None, clone_estimator=True):
+    """Incrementally fit a single binary estimator."""
+    if clone_estimator:
```

Comment:
I'm a bit sceptical because of the special cases handled in ``fit`` but not in ``partial_fit``. Actually I'm not sure what happens in ``fit``. Maybe @arjoly or @mblondel know? I guess this is for the multi-label case with a constant column? When is there a ``-1`` in ``y``?